Pipfile

```
@@ -10,7 +10,7 @@ python_version = "3"
 #
 # NuCypher
 #
-umbral = "==0.1.1a3"
+umbral = "*"
```

We need the current release of Umbral, since the next one will break stuff.

```suggestion
umbral = "==0.1.1a3"
```